src/core_modules/capture-core/components/WidgetProfile/DataEntry/dataEntry.actions.js

```diff
@@ -73,6 +98,68 @@ export const getUpdateFieldActions = (context: Context, innerAction: ReduxAction
         dataEntryActionTypes.UPDATE_FIELD_PROFILE_ACTION_BATCH,
     );
 };
+export const saveSucceed = () => actionCreator(dataEntryActionTypes.UPDATE_SUCCEED)();
+export const saveFailed = () => actionCreator(dataEntryActionTypes.UPDATE_FAILED)();
+export const setTeiModalState = (modalState: string) => actionCreator(dataEntryActionTypes.SET_TEI_MODAL_STATE)({ modalState });
+
+export const updateTeiRequest = ({
+    itemId,
+    dataEntryId,
+    orgUnitId,
+    trackedEntityTypeId,
+    trackedEntityInstanceId,
+    onSaveExternal,
+    onSaveSuccessActionType,
+    onSaveErrorActionType,
+    formFoundation,
+}: {
+    itemId: string,
+    dataEntryId: string,
+    orgUnitId: string,
+    trackedEntityTypeId: string,
+    trackedEntityInstanceId: string,
+    onSaveExternal?: (eventServerValues: any, uid: string) => void,
+    onSaveSuccessActionType?: string,
+    onSaveErrorActionType?: string,
+    formFoundation: RenderFoundation,
+}) =>
+    actionCreator(dataEntryActionTypes.TEI_UPDATE_REQUEST)({
+        itemId,
+        dataEntryId,
+        orgUnitId,
+        trackedEntityTypeId,
+        trackedEntityInstanceId,
+        formFoundation,
+        onSaveExternal,
+        onSaveSuccessActionType,
+        onSaveErrorActionType,
+    });
+
+export const updateTei = ({
+    serverData,
+    onSaveSuccessActionType,
+    onSaveErrorActionType,
+    uid,
+}: {
+    serverData: Object,
+    onSaveSuccessActionType?: string,
+    onSaveErrorActionType?: string,
+    uid: string,
+}) =>
+    actionCreator(dataEntryActionTypes.TEI_UPDATE)(
+        {},
+        {
+            offline: {
+                effect: {
+                    url: 'trackedEntityInstances',
```

Comment:
Reminder to change the URL to the [New tracker endpoints](https://github.com/dhis2/capture-app/pull/2355/files).